docs/zh/react/reactlynx-testing-library/index.mdx

```diff
@@ -137,25 +63,22 @@ it('basic', async function () {
 });
 ```
 
+在这个示例中，你可能已经注意到了我们用到了第三方包 `@testing-library/jest-dom` 中的 `toHaveTextContent` 方法来断言元素的文本内容。这得益于我们底层使用的是 JSDOM 来实现 [Element PAPI](https://lynxjs.org/api/engine/element-api)，因此可以直接使用基于 JSDOM 的 `@testing-library/jest-dom` 中的方法来断言页面元素是否存在。
+
 ### 基础渲染
 
+`render` 方法用于渲染一个 ReactLynx 组件，并返回一个包含 `container`、`rerender`、`unmount`、`getByTestId` 等方法的对象。`container` 是一个包含渲染结果的 LynxElement，`rerender` 用于重新渲染，`unmount` 用于卸载组件、`getByTestId` 用于根据 `data-testid` 属性获取元素。
```

Comment:
`LynxElement` should be code.

Also it's pretty bad to mention each function again and again. You should just link the to `RenderResult` in the API reference